addons/test_website/static/tests/tours/reset_views.js

```diff
@@ -28,10 +28,21 @@ tour.register('test_reset_page_view_complete_flow_part1', {
             // id starting by 'oe_structure..' will actually create an inherited view
             run: "drag_and_drop #oe_structure_test_website_page",
         },
+        {
+            content: "check if the section is dirty",
+            trigger: "#oe_snippets .oe_snippet:has(.s_cover) .oe_snippet_thumbnail:not(.o_we_already_dragging)",
+            run: () => {
+                const wysiwyg = $('#wrapwrap').data('wysiwyg');
+                const node = wysiwyg.editorHelpers.getNodes(document.querySelector('#oe_structure_test_website_page'))[0];
+                if (node.dirty) {
+                    $('#oe_structure_test_website_page').addClass('o_dirty');
+                }
+            },
+        },
```

Comment:
I would think so indeed. @Goaman @sebgeelen Any input on this?

Reply:
Well,
1) the state of being dirty or not is not contained in the DOM anymore so we need to check within the editor if the node is dirty
2) the check of a tour is made by cheking if a selector match rather than an arbitrary function (unless I am missing another possible mechanism)

Therefore I am adding the check of the dirty around the possibilities of the tour.
However we could extends the scope of the test tours mechanisms to support an arbitrary function to be called to check if the step is valid.

Reply:
I just find it strange to see an `if` in a test. Aren't you supposed to _know_ if the node is dirty?

Reply:
> I just find it strange to see an `if` in a test. Aren't you supposed to _know_ if the node is dirty?

He has to add the class only if really dirty so that the test actually tests something... Ok @Goaman then maybe just a missing comment for old dinosaurs like me who wondered why it was necessary to add the `o_dirty` class by hand. And there actually are instances of the o_dirty class in the codebase to remove...

Reply:
> I just find it strange to see an `if` in a test. Aren't you supposed to _know_ if the node is dirty?

You're right, I could have used `toggleClass('o_dirty', node.dirty)`

Reply:
I'm resolving this comment in light of https://github.com/odoo/odoo/pull/60784.